Add some more docs about async commit not used (#5648)
pingcap/docs: system-variables.md

> **Note:**
>
> - If you have enabled TiDB Binlog, enabling this variable cannot improve the performance. To improve the performance, it is recommended to use [TiCDC](/ticdc/ticdc-overview.md) instead.

> > - Enabling this parameter only means that Async Commit becomes an optional transaction commit mode. In fact, it is up to TiDB to choose the most suitable transaction commit mode.

Reviewer: ```suggestion
> - Enabling this parameter only means that Async Commit becomes an optional mode of transaction commit. In fact, the most suitable mode of transaction commit is determined by TiDB.
```

> **Note:**
>
> - If you have enabled TiDB Binlog, enabling this variable cannot improve the performance. To improve the performance, it is recommended to use [TiCDC](/ticdc/ticdc-overview.md) instead.

> > - Enabling this parameter only means that one-phase commit becomes an optional transaction commit mode. In fact, it is up to TiDB to choose the most suitable transaction commit mode.

Reviewer: ```suggestion
> - Enabling this parameter only means that one-phase commit becomes an optional mode of transaction commit. In fact, the most suitable mode of transaction commit is determined by TiDB.
```
